src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorActor.scala

```diff
@@ -122,15 +124,12 @@ class ClusterMonitorActor(val cluster: Cluster,
     * @return ShutdownActor
     */
   private def handleReadyCluster(publicIp: IP): Future[ClusterMonitorMessage] = {
-    for {
+    val result = for {
       // Delete the init bucket
       _ <- deleteInitBucket
       // Remove credentials from instance metadata.
       // Only happens if an notebook service account was used.
       _ <- removeCredentialsFromMetadata
-      // Add Staging Bucket ACLs to the notebook service account.
-      // Only happens if an notebook service account was localized onto the cluster.
-      _ <- gdDAO.setStagingBucketOwnership(cluster)
```

Comment:
Turns out we don't need to do this in the actor anymore, since we're pre-creating the staging bucket with the right permissions (see `BucketService`).